src/test/java/mate/academy/validation/EmailValidatorTest.java

```diff
@@ -0,0 +1,59 @@
+package mate.academy.validation;
+
+import javax.validation.ConstraintValidatorContext;
+import org.junit.jupiter.api.Assertions;
```

Comment:
Static imports for Assertions methods can be used for better readability. Use 'import static org.junit.jupiter.api.Assertions.assertTrue;' and 'import static org.junit.jupiter.api.Assertions.assertFalse;' instead of 'import org.junit.jupiter.api.Assertions;'.